### ISSUE TITLE
Example on README.md don't work

### ISSUE THREAD

**Issue by oxydron** [fchollet/deep-learning-models#37]
numpy is not imported on the example

```python3
from resnet50 import ResNet50
from keras.preprocessing import image
from imagenet_utils import preprocess_input, decode_predictions

model = ResNet50(weights='imagenet')

img_path = 'elephant.jpg'
img = image.load_img(img_path, target_size=(224, 224))
x = image.img_to_array(img)
x = np.expand_dims(x, axis=0)
x = preprocess_input(x)

preds = model.predict(x)
print('Predicted:', decode_predictions(preds))
```

`import numpy as np` should solve the problem.

**Comment by oxydron** [comment]
Another thing: this code doesn't work with python 3.

**Comment by gugarosa** [comment]
Just use the whole file instead of only the example: https://github.com/fchollet/deep-learning-models/blob/master/resnet50.py

And I can confirm it is working with python3 as I'm using it and had no problems so far.

**Comment by NerminSalem** [comment]
from vgg19 import VGG19
from keras.preprocessing import image
from imagenet_utils import preprocess_input
from keras.models import Model
import numpy as np

base_model = VGG19(weights='imagenet')
model = Model(input=base_model.input, output=base_model.get_layer('block4_pool').output)

img_path = 'elephant.jpg'
img = image.load_img(img_path, target_size=(224, 224))
x = image.img_to_array(img)
x = np.expand_dims(x, axis=0)
x = preprocess_input(x)

block4_pool_features = model.predict(x)
also doesn't work?

**Comment by NerminSalem** [comment]
i really need help on how to extract features using vgg19?
@oxydron @fchollet @joeyhng  @tdeboissiere
